package/python-kiwi-spec-template

```diff
@@ -641,7 +639,12 @@ Group:          %{sysgroup}
 Provides manual pages to describe the kiwi commands
 
 %prep
-%autosetup -p1 -n kiwi-%{version}
+%setup -q -n kiwi-%{version}
+
+%if 0%{?suse_version}
+# Apply SUSE specific patches
```

Comment:
Maybe you want to replace the comment from the one in L61, to avoid the repetition of being SUSE specific.